app/controllers/provider_interface/provider_relationship_permissions_controller.rb

```diff
@@ -61,9 +68,10 @@ def render_404_unless_permissions_found
     end
 
     def render_403_unless_access_permitted
-      render_403 unless current_provider_user.providers.include?(
-        training_provider_permissions.training_provider,
-      )
+      render_403 unless current_provider_user.can_manage_organisations? &&
```

Comment:
Will this also return true in the event that the user has manage_orgs permission for a provider that isn't the current training provider?

Reply:
Yes sorry as @mnacos pointed out this should be scoped to a particular provider. I'll add a method to `ProviderAuthorization` for this.

Reply:
https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/2231/commits/f062678cfbf7db0d391234cf4d9b0367db0e0d95 and the commit before it https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/2231/commits/2f8bc6a1bb687e12f8c31d364597d570e05e625a use a more specific scope to check the permissions now.